ParametricObjectCreation/honeycomb_solid/honeycomb_solid.py

```diff
@@ -163,7 +181,7 @@ def getDefaultDisplayMode(self):
         It must be defined in getDisplayModes.
 
         """
-        return 'Shaded'
+        return "Shaded"
```

Comment:
Single-quotes are preferred in Python.

Reply:
Actually, I ran the code through ``black`` and it turned those single quotes two double quotes, as it is a very reputable linter that follows PEP8 I believe it is double quotes that is preferred. But, if this is against FreeCAD’s Python convention, I can revert them back to single quotes.